src/ripple/protocol/nft.h

```diff
@@ -54,6 +54,7 @@ constexpr std::uint16_t const flagBurnable = 0x0001;
 constexpr std::uint16_t const flagOnlyXRP = 0x0002;
 constexpr std::uint16_t const flagCreateTrustLines = 0x0004;
 constexpr std::uint16_t const flagTransferable = 0x0008;
+constexpr std::uint16_t const flagMutable = 0x0016;
```

Comment:
The number is in hex
```suggestion
constexpr std::uint16_t const flagMutable = 0x0010;
```

Reply:
It seems the flag numbers increment by a power of 2 (in hex). Is this the case, and if so, what's the reasoning for it?